src/app/votacao/publica/convidados/page.js

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import Navbar from "@/components/navbar";
+import Footer from "@/components/footer";
+import { Image } from 'lucide-react';
+
+export default function VotacaoPublica() {
+    return (
+        <div className="flex flex-col p-0 w-full min-h-screen">
+            <Navbar />
+            <main className="flex flex-col flex-grow items-center justify-start">
+                <div className="flex flex-col w-5/6 mt-4 md:mt-16">
+                    <p className="font-sans font-regular text-left w-full text-[15px] md:text-[25px] text-[#004854] mb-[-8px] md:mb-[-16px]">
+                        Escolha de
+                    </p>
+                    <p className="font-sans font-bold text-left w-full text-[23px] md:text-[36px] text-[#004854]">
+                        Projeto
+                    </p>
+                    <hr className="w-full border-[#004854] border-1 " />
+                    <div className="flex flex-col md:flex-row md:w-5/6 gap-8 mt-6 md:gap-12 md:mt-16 md:mb-16 md:justify-stretch md:self-center">
+                        <div className="order-2 w-170px w-4/5 md:w-1/3 h-fit bg-[#ffffff] rounded-b-[10px] flex flex-col shadow-gray-400 shadow-md self-center">
+                            <div className="w-full aspect-video bg-[#cdcdcd] border-none p-0 place-content-center">
+                                <Image
+                                    size={50}
+                                    className="justify-self-center stroke-white"
+                                />
```

Comment:
Nesta parte o ESlint esta reclamando pois é uma boa pratica colocar alt nas imagens, sugestão de como por
```suggestion
                                <Image
                                    size={50}
                                    className="justify-self-center stroke-white"
                                    alt="Imagem do Projeto"
                                />
```

Reply:
OK, já vou alterar.